shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/framework/container/database/MySQLContainer.java

```diff
@@ -44,10 +45,10 @@ protected void configure() {
         withClasspathResourceMapping("/env/mysql/my.cnf", "/etc/mysql/my.cnf", BindMode.READ_ONLY);
         super.configure();
         withExposedPorts(getPort());
-        setWaitStrategy(new LogMessageWaitStrategy().withRegEx(".*ready for connections.*"));
         if (IntegrationTestEnvironment.getInstance().getItEnvType() == ITEnvTypeEnum.LOCAL) {
             addFixedExposedPort(3306, 3306);
         }
+        setWaitStrategy(new JDBCConnectionWaitStrategy(() -> DriverManager.getConnection(DataSourceEnvironment.getURL(DATABASE_TYPE, "localhost", getFirstMappedPort()), "root", "root")));
```

Comment:
It's better to close connection.

Reply:
<img width="703" alt="image" src="https://user-images.githubusercontent.com/101622833/166130563-b8ed92ea-a91f-4f97-99e0-8014d83e85c4.png">
It's seems will auto close connection.